Templates/Templates/AutoDecodable.stencil

```diff
@@ -0,0 +1,96 @@
+import Foundation
+
+extension KeyedDecodingContainer {
+  public func decodeOrDefault<T: Decodable>(_ type: T.Type, forKey key: Key, defaultValue: T) throws -> T {
+    guard let value: T = try decodeIfPresent(type, forKey: key) else {
```

Comment:
If there is default value then, I think, there is no need to throw when key is not there and default value should be applied, so it should be `try?` and method should not throw any error. I ended up using pair of `decodeSafely` and `decodeSafelyIfPresent` similar to described here https://kean.github.io/post/codable-tips-and-tricks